### PR TITLE
SSL support

### LINKED ISSUE #7: HTTPS support
While the android app supports `https`, `http` is hardcoded in this library.
I would love to submit a PR to support that. Do you still maintain this library?

### PR DESCRIPTION
**Pull request by markusressel**
added ssl parameter
added default port

It is noteworthy that although the android app supports https, the certificate that is used is self signed, so when connecting one would have to disable certificate checks in python f.ex. like this:

```
async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(ssl=False)) as session:
  [...]
```

fixes #7

### CODE REVIEW

File: pydroid_ipcam.py [hunk 1]
```diff
@@ -11,30 +11,35 @@
...
+DEFAULT_SSL = True
+
 
 class PyDroidIPCam:
     """The Android device running IP Webcam."""
 
-    def __init__(self, loop, websession, host, port, username=None,
-                 password=None, timeout=10):
+    def __init__(self, loop, websession, host, port=DEFAULT_PORT, username=None,
+                 password=None, timeout=10, ssl=DEFAULT_SSL):
```
Comment (pvizeli) [inline pydroid_ipcam.py]: Please use `port=8080, ..., ssl=True`

File: pydroid_ipcam.py [hunk 2]
```diff
@@ -11,30 +11,35 @@
...
         """Initialize the data oject."""
         self.loop = loop
         self.websession = websession
         self.status_data = None
         self.sensor_data = None
         self._host = host
-        self._port = port
+        self._port = port if port is not None else DEFAULT_PORT
```
Comment (pvizeli) [inline pydroid_ipcam.py]: Is not needed, because it is not our trouble if someone call this with None

File: pydroid_ipcam.py [hunk 3]
```diff
@@ -11,30 +11,35 @@
...
 
         if username and password:
             self._auth = aiohttp.BasicAuth(username, password=password)
 
     @property
     def base_url(self):
         """Return the base url for endpoints."""
-        return "http://{}:{}".format(self._host, self._port)
+        protocol = "https" if self._ssl else "http"
+        return "{}://{}:{}".format(protocol, self._host, self._port)
```
Comment (pvizeli) [inline pydroid_ipcam.py]: We can now use f strings: `f"{protocol}://{host}:{port}"`

Comment (markusressel) [reply pydroid_ipcam.py]: I changed all .format() instances too, hope it's in your interest.

**Pull request merged**